lib/components/base/DropdownSelect.vue

```diff
@@ -163,8 +192,17 @@ export default {
       transition: filter 0.3s ease-in-out;
     }
 
-    &.dropdown-open {
-      border-radius: var(--radius-md) var(--radius-md) 0 0;
+    &.disabled {
+      cursor: not-allowed;
+      filter: brightness(0.75);
+    }
+
+    &.render-up {
+      border-radius: 0 0 var(--radius-lg) var(--radius-lg);
```

Comment:
The border radius should be --radius-md to be consistent with above (and in other places)